dynamiq/connections/connections.py

```diff
@@ -1232,3 +1233,62 @@ def conn_params(self) -> dict:
             "api_base": self.url,
             "api_key": self.api_key,
         }
+
+
+class MPC(BaseConnection):
+    # SSE
+    url: str | None = Field(None, description="The SSE endpoint URL to connect to.")
+    headers: dict[str, Any] | None = Field(default=None, description="Optional headers to include in the SSE request.")
+    timeout: float = Field(default=5.0, description="Timeout in seconds for establishing the initial connection.")
+    sse_read_timeout: float = Field(
+        default=60 * 5, description="Timeout in seconds for reading SSE messages. Defaults to 5 minutes."
+    )
+
+    # STDIO
+    command: str | None = Field(None, description="The executable to run to start the server.")
+    args: list[str] = Field(default_factory=list, description="Command line arguments to pass to the executable.")
+    env: dict[str, str] | None = Field(None, description=" The environment to use when spawning the process.")
+    cwd: str | Path | None = Field(None, description="The working directory to use when spawning the process.")
+    encoding: str = Field(
+        default="utf-8", description="The text encoding used when sending/receiving messages to the server."
+    )
+    encoding_error_handler: Literal["strict", "ignore", "replace"] = Field(
+        default="strict", description="The text encoding error handler."
+    )
+
+    def connect(self):
+        pass
+
+    async def get_client(self):
```

Comment:
It seems this logic returns just client, so it could be moved to `def connect` as it done in the rest connections